.github/workflows/pr.yaml

```diff
@@ -124,7 +98,7 @@ jobs:
     needs: [conda-cpp-build, changed-files]
     secrets: inherit
     uses: rapidsai/shared-workflows/.github/workflows/conda-cpp-tests.yaml@branch-24.10
-    if: needs.changed-files.outputs.test_cpp == 'true'
+    if: fromJSON(needs.changed-files.outputs.transformed_output).test_cpp
```

Comment:
Let's consider alternative names for `transformed_output`. It feels like it's exposing too much implementation detail from the shared action.

Maybe `changed_groups`?

Reply:
I changed it to `changed_file_groups`.